pkg/kubernetes/scripts/volumes.js

```diff
@@ -872,6 +907,34 @@
                 return defer.promise;
             }
 
+            function updatePVC() {
+                /*
+                 * HACK: Until the API gets a way to manually
```

Comment:
There's this bug, https://github.com/kubernetes/kubernetes/issues/21498. Basically this code being rewritten right now in kubernetes, Once the refactor lands I'll reevaluate and see if further bugs need to be opened.